sync-teams-to-gh.py

```diff
@@ -260,14 +260,14 @@ def http_method(url, data={}, method=None, fail_ok=False):
 
         gh_role = response.get("role_name")
 
-        # The GitHub API has inconsistent role values, so we need to translate.
-        # https://github.com/github/rest-api-description/issues/1378
-        gh_role = {
-            "write": "push",
-            "read": "pull"
-        }.get(gh_role, gh_role)
-
         if gh_role != role:
+            # The GitHub API has inconsistent role values, so we need to translate.
```

Comment:
Should we update role before checking if ` gh_role != role`?

Reply:
This requires role to use `read/write`, like the values downloaded from GH.